web/routing.go

```diff
@@ -210,14 +212,16 @@ func SetupRoutes(router *echo.Echo, services *stack.Services) error {
 	{
 		mwsNotBlocked := []echo.MiddlewareFunc{
 			middlewares.NeedInstance,
+			middlewares.NeedFlags,
```

Comment:
NO!!!

It would slow down every request to the stack for fetching the flags, while only very few needs them.